components/Navigation/NavigationBar.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import NavigationItem from "./NavigationItem";
+import HomeIcon from "../icons/HomeIcon";
+import WordsIcon from "../icons/WordsIcon";
+import DashboardIcon from "../icons/DashboardIcon";
+
+const ICON_WIDTH = "1.5rem";
+
+export default function NavigationBar() {
+  return (
+    <nav>
+      <NavigationItems>
+        <NavigationItem
+          path={"/"}
+          pageName={"Home"}
```

Comment:
```suggestion
          path="/"
          pageName="Home"
```
you don't need to wrap strings in curly braces.